dimagi/pages/static/style/pages/sectors/nutrition_programs.scss

```diff
@@ -0,0 +1,117 @@
+@import 'app/mixins/background';
+@import 'app/variables/colors';
+@import 'app/variables/sizes';
+@import 'app/variables/type';
+
+#page-intro {
+  $_base-height: 190px;
+  $_height-by-size: (
+    small: 280px,
+    medium: 280px,
+    large: 280px,
+    extra-large: 340px,
+  );
+  @include make-bg(hero-data-image, '../../../pages/sectors/nutrition/intro/hero', $_base-height, $_height-by-size);
+}
+
+.news-images .image.blog1 {
+  @include background-cover('../../../pages/sectors/nutrition/news/Blog1.jpg');
```

Comment:
please see the file naming conventions comment i made in https://github.com/dimagi/dimagi.com/pull/92

Reply:
no capitalization in the image filenames and no CamelCase